api/src/main/java/org/apache/iceberg/expressions/Zorder.java

```diff
@@ -17,24 +17,19 @@
  * under the License.
  */
 
-package org.apache.spark.sql.catalyst.plans.logical
+package org.apache.iceberg.expressions;
 
-import org.apache.iceberg.NullOrder
-import org.apache.iceberg.Schema
-import org.apache.iceberg.SortDirection
-import org.apache.iceberg.SortOrder
-import org.apache.iceberg.expressions.Term
+import java.util.Arrays;
+import java.util.List;
 
-class SortOrderParserUtil {
+public class Zorder implements Term {
```

Comment:
Thinking about this a bit more, I think we should move this out of the API module. We don't want to make guarantees about this class since it isn't a "real" part of the expression library yet. It is mostly a placeholder. Can you move this to core?

Reply:
moved.